docs: add URLs to Prisma database section (#1161)
t3-oss/create-t3-app: www/src/pages/en/usage/first-steps.md

## Database

> If your app includes Prisma, make sure to run `npx prisma db push` from the root directory of your app. This command will sync your Prisma schema with your database and will generate the TypeScript types for the Prisma Client based on your schema. Note that you need to [restart the TypeScript server](https://stackoverflow.com/a/64456017) after doing this so that it can detect the generated types.

Reviewer: ```suggestion
If your app includes Prisma, make sure to run `npx prisma db push` from the root directory of your app. This command will sync your Prisma schema with your database and will generate the TypeScript types for the Prisma Client based on your schema. Note that you need to [restart the TypeScript server](https://tinytip.co/tips/vscode-restart-ts/) after doing this so that it can detect the generated types.
```

You should now be able to log in.

## Editor Setup

The following extensions are recommended for an optimal developer experience. The links below provide editor specific plugin support.

- [Prisma Extension](https://www.prisma.io/docs/guides/development-environment/editor-setup)

> - [Tailwind CSS IntelliSense plugin](https://tailwindcss.com/docs/editor-setup)

Reviewer: ```suggestion
- [Tailwind CSS IntelliSense Extension](https://tailwindcss.com/docs/editor-setup)
```

?

> - [Prettier Extension ](https://prettier.io/docs/en/editors.html)

Reviewer 2: extra space here
```suggestion
- [Prettier Extension](https://prettier.io/docs/en/editors.html)
```
